go/ct/spc/specification_test.go

```diff
@@ -69,13 +69,13 @@ func TestSpecification_SpecificationIsComplete(t *testing.T) {
 }
 
 func TestSpecification_EachRuleProducesAMatchingTestCase(t *testing.T) {
+	rnd := rand.New(0)
```

Comment:
:eyes: follow-up issue? Any thought about why this happens?

Reply:
Is `rnd` actually thread save? The tests are run in parallel.

Reply:
This will be investigated in #550.